app/views/shared/mailers/_social_and_contact.html.haml

```diff
@@ -17,6 +17,9 @@
               - if ContentConfig.footer_linkedin_url.present?
                 %a.soc-btn.li{href: ContentConfig.footer_linkedin_url}
                   LinkedIn
+              -if ContentConfig.footer_instagram_url.present?
+                %a.soc-btn.tw{href: ContentConfig.footer_instagram_url}
```

Comment:
You are re-using the twitter colour here by using the `tw` css class. I think that's a good solution for now. Ideally we would come up with another colour that fits into the design but it's not worth the effort and should probably be up to a designer. It can also be done in another issue if anyone ever complains.